azdev/commands.py

```diff
@@ -30,10 +30,14 @@ def operation_group(name):
     with CommandGroup(self, 'statistics', operation_group('statistics')) as g:
         g.command('list-command-table', 'list_command_table')
         g.command('diff-command-tables', 'diff_command_tables')
+        g.command('gen-command-meta', 'gen_command_meta')
 
     with CommandGroup(self, 'verify', operation_group('pypi')) as g:
         g.command('history', 'check_history')
 
+    with CommandGroup(self, 'checker', operation_group('checker')) as g:
```

Comment:
use `breaking-change`?

Reply:
Or you can create a command group called "statistics command-metadata", it has two sub commands:
- "statistics command-metadata generate"
- "statistics command-metadata diff"